src/ai/ai.cpp

```diff
@@ -1179,14 +1200,24 @@ void take_reforms(sys::state& state) {
 			continue;
 
 		if(n.get_is_civilized()) { // political & social
-			float max_support = 0.0f;
+			// Enact social policies to deter Jacobin rebels from overruning the country
+			// Reactionaries will popup in effect but they are MORE weak that Jacobins
```

Comment:
The reason this looked at movements before is that, if you don't pick the issue most supported by an ongoing movement, everyone in every other movement will get angry. I am willing to try your new version instead, but keep that in mind. You may eventually need to take both into account